### DISCUSSION TITLE
Little change mainwindows view

### DISCUSSION THREAD

**Discussion by ksmagala** [collin80/SavvyCAN#538] [Q&A]
Hello, I want to make some view changes in mainwindow.ui. I want to split canFramesView into CAN 0 and CAN 1 and add "Send Frames" under canFramesView. I tried to wrap it in a vertical layout, but the application does not correctly arrange the view when launched. The window app is not responsive when maximalized. Can you explain an easy way to do this or give me a tip?

**Comment by collin80** [comment]
Well, what you want to do should be possible. But, QT can be kind of tough sometimes when you are creating GUI layouts. The "central widget" of the main screen is set to horizontal layout so if you want to stack two views in a vertical layout you would probably have to add a vertical layout inside the horizontal layout then put your CAN0 and CAN1 views inside that vertical layout. After a while you end up with a big ball of layouts inside layouts but it can work.
